src/main/pages/che-7/administration-guide/proc_assigning-che-permissions.adoc

```diff
@@ -22,10 +22,10 @@ The following is a message body that requests permissions for a user with a `use
     "configure",
     "setPermissions"
   ],
-  "userId": "userID",          <1>
+  "userID": "userID",          <1>
```

Comment:
This is a config file, so I'm pretty sure you cannot change the name of the param. Check with SMEs, please.